app/assets/javascripts/Components/markdown_editor.jsx

```diff
@@ -69,10 +49,13 @@ export default class MarkdownEdit extends React.Component {
   }
 }
 
-MarkdownEdit.propTypes = {
+MarkdownEditor.propTypes = {
   annotation_text_id: PropTypes.string,
   content: PropTypes.string.isRequired,
   handleChange: PropTypes.func.isRequired,
   show_autocomplete: PropTypes.bool,
+  text_area_id: PropTypes.string,
+  auto_completion_text_id: PropTypes.string,
+  auto_completion_list_id: PropTypes.string,
   updateAnnotationCompletion: PropTypes.func,
 };
```

Comment:
I think it could be worth adding some documentation to clarify when the props are mandatory if show_autocomplete is on. What do you think?